### ISSUE TITLE
EV3 serial console - no input

### ISSUE THREAD

**Issue by anicolao** [ev3dev/ev3dev#12]
## This issue is a transcript of an email thread between Ralph and I in case we find a solution that's useful to others.

tl;dr - output works on serial console, input doesn't, need a clue.

Long version:

Sorry to bother you but I was trying to hack on my EV3 and as far as I can tell you're one of very few competent people working on the device.

Both with the stock distribution and with your ev3dev distribution, I can see output on the serial console and type, but the system doesn't react. Characters are echoed back across the connection but that's it, I can't log in or interact with a shell (after changing /etc/inittab to spawn a shell instead of anything else at startup) or find any stty settings that make any difference (after changing /etc/inittab to start a shell script that messes with the terminal before starting a shell).

So I wanted to confirm - in your ev3dev setup, you're able to actually work on the serial console, right?

Any insight much appreciated,
